samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
I'm also working on another language package for Atom, which is for Unix manual pages. It'll actually support the entire troff typesetting language, as opposed to just a handful of macros. Biggest improvement to the existing man-page grammar is the existence of markup classes... so hopefully source code on GitHub will be a lot more readable with bold/italic/etc styling applied. =)